src/js/videojs-dash.js

```diff
@@ -80,6 +80,13 @@ class Html5DashJS {
       this.mediaPlayer_.setLimitBitrateByPortal(false);
     }
 
+    this.mediaPlayer_.on(dashjs.MediaPlayer.events.ERROR, function(event) {
+      this.tech_.error({
```

Comment:
Do we want to error on all dash.js errors (including timed text issues, download errors, etc.)?